StudyHub/StudyHub.BLL/Services/Interfaces/IUserService.cs

```diff
@@ -9,4 +9,5 @@ public interface IUserService
 {
     Task<PageList<StudentDTO>> GetStudentsAsync(SearchRequest request);
     Task<UserDTO> UpdateUserAsync(Guid userId, UpdateUserDTO dto);
+    Task<UserDTO> GetUserAsync(Guid userId);
```

Comment:
```suggestion
    Task<UserDTO> Me(Guid userId);
```

Reply:
Why? You creat this service for profile. So i think it is better to call it "Me" for this service

Reply:
1) where async?
2)we called method what it doing(especially in service). You calling method Me, what does he doing? Its clear to us what it is, for other developers - not

Reply:
U get information about user only for profile

Reply:
I think service method named "me" doesn't clear define what method is actually doing. Keep it like GetUserById or smth like it was